spark/src/main/scala/org/apache/spark/sql/comet/CometNativeScanExec.scala

```diff
@@ -71,6 +72,85 @@ case class CometNativeScanExec(
   }
 
   override def hashCode(): Int = Objects.hashCode(output)
+
+  override lazy val metrics: Map[String, SQLMetric] = {
+    CometMetricNode.baselineMetrics(sparkContext) ++
+      Map(
```

Comment:
Should we have some way of distinguishing between these metrics and those from the current native scan? Perhaps the display string can have a short prefix?

Reply:
+1. We need not do this for all operators. This is just so we could distinguish between metrics reported by different scan implementations.